guard/src/commands/validate.rs

```diff
@@ -56,6 +54,15 @@ pub(crate) enum Type {
     Generic,
 }
 
+impl From<&str> for Type {
```

Comment:
Need to understand why is this additional snippet required? Is it because older version of Clap seamlessly converted strings to enums but newer one doesn't?

Reply:
no it didnt, it still treated it as string and then matched on it inside the execute function. The logic is the same, but this is just a more idiomatic way to handle conversions in rust